src/components/EndpointExplorer.js

```diff
@@ -102,7 +102,18 @@ function buildRequestUrl (baseUrl, endpoint, values) {
   _.each(template.varNames, (varName) => {
     let objectPath = (varName in endpoint.path) ?
       endpoint.path[varName] : varName + '.value';
-    let value = _.get(values, objectPath);
+    let value;
+
+    if (typeof objectPath === 'string') {
+      value = _.get(values, objectPath);
+    } else if (typeof objectPath === 'function') {
+      try {
+        value = objectPath(values);
+      } catch (e) {
+        console.log(`Extraction function in ${endpoint.label} path.${varName} failed`)
```

Comment:
Why silently failing?  If `objectPath` throws an error, shouldn't that be handled in some way other than simply printing a message to the developer console and continuing on?